### PR TITLE
Add `dydx-testnet-4` folder and update workflows

### PR DESCRIPTION
**Pull request by ttl33**

### CODE REVIEW

File: .github/workflows/validate_gentx.yml [hunk 1]
```diff
@@ -18,4 +18,4 @@ jobs:
         with:
           name: genesis.new.json
           path: dydx-testnet-3/genesis.new.json
-          if-no-files-found: error
\ No newline at end of file
+          if-no-files-found: -4
```
Comment (topperdogger123) [inline .github/workflows/validate_gentx.yml:21]: Did you mean to change this?

File: .github/workflows/validate_final_genesis.yml [hunk 2]
```diff
@@ -1,20 +1,20 @@
...
 
 jobs:
-  dydx-testnet-3:
+  dydx-testnet-4:
     runs-on: ubuntu-latest
     steps:
       - name: Git checkout
         uses: actions/checkout@v3
       - name: Validate final genesis file
-        run: make validate-dydx-testnet-3-final-genesis
```
Comment (topperdogger123) [inline .github/workflows/validate_final_genesis.yml:20]: You'll also need to change the commands in Makefile.

File: .github/workflows/validate_final_genesis.yml [hunk 3]
```diff
@@ -1,20 +1,20 @@
...
+        run: make validate-dydx-testnet-4-final-genesis
```
Comment (roy-dydx) [inline .github/workflows/validate_final_genesis.yml:20]: You'll also need to change these in Makefile

Comment (ttl33) [reply .github/workflows/validate_final_genesis.yml:20]: yup, good catch, missed that

File: dydx-testnet-4/gentx/dummy.json [hunk 4]
```diff
@@ -0,0 +1 @@
+{}
```
Comment (topperdogger123) [inline dydx-testnet-4/gentx/dummy.json:1]: Probably remove this, it might cause problems

Comment (roy-dydx) [inline dydx-testnet-4/gentx/dummy.json:1]: Probably remove this, it might cause problems

**Comment by ttl33** [comment]
Closing this PR for two reasons:
1. Without `pregenesis.json`, new `gentx` does NOT get validated. So, I tried adding the `pregenesis.json` but I believe we are editing the `dv4nt` -> `adv4nt` so I think it's better to wait for @teddyding 's changes
2. I tried adding a dummy file in `dydx-testnet-4/` folder to initialize the folder, but the workflow expects to see at least one `gentx`

**Pull request closed without merging**